Detectors/CTP/workflow/src/RawDecoderSpec.cxx

```diff
@@ -81,6 +81,12 @@ void RawDecoderSpec::run(framework::ProcessingContext& ctx)
     auto triggerOrbit = o2::raw::RDHUtils::getTriggerOrbit(rdh);
     uint32_t stopBit = o2::raw::RDHUtils::getStop(rdh);
     uint32_t packetCounter = o2::raw::RDHUtils::getPageCounter(rdh);
+    uint32_t version = o2::raw::RDHUtils::getVersion(rdh);
+    if (version == 7) {
+      mPadding = 0;
+    }
```

Comment:
you can simply do `mPadding = o2::raw::RDHUtils::getDataFormat(rdh) == 0`, w/o checking the version (will be done internally).